src/components/FormPairRow.tsx

```diff
@@ -122,7 +122,7 @@ const FormPairRow = (props: IPairRow) => {
 
   const usdValueDisplay =
     usdValue && usdValue.gt(0.01) // If smaller than 0.01 cents, dont show
```

Comment:
If exactly 0.01, do we wanna show it too? Should we change it to gte?

Reply:
this is done